microsoft-advanced-threat-analytics/info.json

```diff
@@ -0,0 +1,144 @@
+{
+  "name": "microsoft-advanced-threat-analytics",
+  "label": "Microsoft Advanced Threat Analytics",
+  "version": "1.0.0",
+  "description": "Microsoft Advanced Threat Analytics (ATA) is an on-premises platform that helps protect your enterprise from multiple types of advanced targeted cyber attacks and insider threats.",
+  "publisher": "Fortinet",
+  "cs_approved": false,
+  "cs_compatible": true,
+  "category": "Threat Detection",
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "help_online": "",
+  "configuration": {
+    "fields": [
+      {
+        "name": "url",
+        "title": "Server URL",
+        "type": "text",
+        "editable": true,
+        "visible": true,
+        "required": true,
+        "description": "Specify the Microsoft Advanced Threat Analytics server URL to connect and perform the automated operations.",
+        "placeholder": "https://atacenter.contoso.com"
+      },
+      {
+        "name": "username",
+        "title": "Username",
+        "type": "text",
+        "editable": true,
+        "visible": true,
+        "required": true,
+        "description": "Specify the Microsoft Advanced Threat Analytics username to connect and perform the automated operations."
+      },
+      {
+        "name": "password",
+        "title": "Password",
+        "type": "password",
+        "editable": true,
+        "visible": true,
+        "required": true,
+        "description": "Specify the Microsoft Advanced Threat Analytics password to connect and perform the automated operations.."
+      },
+      {
+        "name": "verify_ssl",
+        "title": "Verify SSL",
+        "type": "checkbox",
+        "editable": true,
+        "visible": true,
+        "required": false,
+        "value": true,
+        "description": "Specifies whether the SSL certificate for the server is to be verified. \nBy default, this option is set to True."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "get_suspicious_activities_list",
+      "title": "Get Suspicious Activities List",
+      "annotation": "get_suspicious_activities_list",
+      "description": "Retrieves the suspicious activities by the activity ID, status, severity from Microsoft Advanced Threat Analytics.",
+      "category": "investigation",
+      "visible": true,
+      "enabled": true,
+      "parameters": [
+        {
+          "name": "activity_id",
+          "title": "Suspicious Activity ID",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the ID of the suspicious activity.",
+          "description": "Specify the ID of the suspicious activity to retrieve. If provided, all other arguments will be ignored."
```

Comment:
There is only one parameter and it is required . Remove `If provided, all other arguments will be ignored.` or provide other parameters